### ISSUE TITLE
take_screenshot misses . in filename, so can't view

### ISSUE THREAD

**Issue by LukasWallrich** [ashbythorpe/selenider#2]
Thank you for this great package that I am very excited to explore.

I just ran into a minor issue that I wanted to report - in the below it seems that a `.` is missing in front of the file extension.

Probably `withr::local_tempfile(fileext = "png")` should be `withr::local_tempfile(fileext = ".png")`?

```
selenider::take_screenshot(view = T)
[1] "/var/folders/xk/g0cqx1hs53z_txqsyq74jzcc0000gn/T//Rtmpc9PUbI/file470d46107fdfpng"
Warning message:
In onRejected(reason) :
  An error occurred: Error in showimage::show_image(filename): Unknown image type
```

**Comment by ashbythorpe** [comment]
This should be fixed in the development version of selenider (`remotes::install_github("ashbythorpe/selenider")`). Let me know if it is working for you.

There probably won't be a CRAN release too soon, because I'm currently working on an [alternative to RSelenium](https://github.com/ashbythorpe/selenium-r), that I want to integrate into this package.

**Comment by ashbythorpe** [comment]
This change is now released in selenider 0.2.0
